packages/@aws-cdk/aws-cloud9/test/integ.cloud9.ts

```diff
@@ -0,0 +1,39 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as cdk from '@aws-cdk/core';
+import * as cloud9 from '../lib';
+
+export class Cloud9Env extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    const vpc = new ec2.Vpc(this, 'VPC', {
+      maxAzs: 2,
+      natGateways: 1
+    });
+
+    // create a cloud9 ec2 environment in a new VPC
+    const c9env = new cloud9.EnvironmentEc2(this, 'C9Env', { vpc });
+
+    new cdk.CfnOutput(this, 'URL', { value: c9env.ideUrl });
+    new cdk.CfnOutput(this, 'ARN', { value: c9env.environmentEc2Arn });
+
+    // create the cloud9 environment in my default VPC
+    const vpc2 = ec2.Vpc.fromLookup(this, 'VPC2', { isDefault: true });
```

Comment:
What's with the 2 VPCs in this test? I would just use `vpc`, and get rid of `vpc2`.